x/evm/types/testutils/rand.go

```diff
@@ -129,7 +129,7 @@ func getConfirmedEventQueue(cdc codec.Codec, events []types.Event) utils.QueueSt
 		}
 
 		qs.Items[fmt.Sprintf("%s_%s", queueName, event.GetID())] = utils.QueueState_Item{
-			Key:   eventPrefix.AppendStr(string(event.GetID())).AsKey(),
+			Key:   eventPrefix.Append(utils.LowerCaseKey(string(event.GetID()))).AsKey(),
```

Comment:
why does it differ? shouldn't event.GetID() always be lowercase?

Reply:
The other one doesn't matter for the purpose of testing. I don't understand how this was not broken before though.